tests/test_cmd_cache_create.py

```diff
@@ -15,43 +15,205 @@
 BINARY_CONTENT_1 = bytes([0x01, 0x02, 0x03, 0x04])
 BINARY_CONTENT_2 = bytes([0x08, 0x09, 0x0A, 0x0B, 0x0C, 0x0D, 0x0E, 0x0F, 0x10, 0x11])
 
-URI1 = "#first" # [0x23, 0x66, 0x69, 0x72, 0x73, 0x74]
-URI2 = "#second" # [0x23, 0x73, 0x65, 0x63, 0x6F, 0x6E, 0x64]
+URI1 = "#first"  # [0x23, 0x66, 0x69, 0x72, 0x73, 0x74]
+URI2 = "#second"  # [0x23, 0x73, 0x65, 0x63, 0x6F, 0x6E, 0x64]
 
-EXPECTED_CACHE_EB_8 = bytes([0xBF,
-                             0x66, 0x23, 0x66, 0x69, 0x72, 0x73, 0x74, # tstr "first"
-                             0x5A, 0x00, 0x00, 0x00, 0x04, # bstr size 4
-                             0x01, 0x02, 0x03, 0x04, # BINARY_CONTENT_1
-                             0x60, 0x45, 0x00, 0x00, 0x00, 0x00, 0x00, # padding
-                             0x67, 0x23, 0x73, 0x65, 0x63, 0x6F, 0x6E, 0x64, # tstr "second"
-                             0x5A, 0x00, 0x00, 0x00, 0x0a, # bstr size 10
-                             0x08, 0x09, 0x0A, 0x0B, 0x0C, 0x0D, 0x0E, 0x0F, 0x10, 0x11, # BINARY_CONTENT_2
-                             0x60, 0x47, 0x00, 0x00, 0x00, 0x00, 0x00, 0x00, 0x00, # padding
-                             0xFF
-                             ])
+EXPECTED_CACHE_EB_8 = bytes(
```

Comment:
Personally - preferred the previous formatting. Can we force black to ignore this section?

Reply:
I agree, following solution can be used:
https://black.readthedocs.io/en/stable/usage_and_configuration/the_basics.html#ignoring-sections

Reply:
I agree, new formatting is very ugly - previous version will be restored.